Code/GraphMol/SmilesParse/CXSmilesOps.cpp

```diff
@@ -414,6 +414,70 @@ std::string quote_string(const std::string &txt) {
   // FIX
   return txt;
 }
+
+std::string get_enhanced_stereo_block(const ROMol &mol,
+                            const std::vector<unsigned int> &atomOrder){
+  std::string res = "";
```

Comment:
A stringstream, maybe? Just a personal preference.

Reply:
could do that.